src/constants.js

```diff
@@ -65,3 +65,5 @@ export const HATHOR_TOKEN_CONFIG = {'name': 'Hathor', 'symbol': 'HTR', 'uid': '0
  * Hathor token default index
  */
 export const HATHOR_TOKEN_INDEX = 0;
+
+export const GTM_ID = process.env.REACT_APP_GTM_ID;
```

Comment:
I think it's becoming difficult to know exactly which env vars this project expects. Maybe we should at least keep them all together in this file (close to each other).

But the main thing here is that now this env var is required, right?

But for development, one shouldn't have to define this to make the project run in his/her machine. We should have some mechanism in place to make this env var optional for development.